WordPress/Classes/ViewRelated/System/Floating Create Button/SheetActions.swift

```diff
@@ -75,4 +75,8 @@ struct StoryAction: ActionSheetItem {
         badge.backgroundColor = Constants.Badge.backgroundColor
         return badge
     }
+
+    static func shouldBeShown(blog: Blog?) -> Bool {
+        return (blog?.supports(.stories) ?? false) && !UIDevice.isPad() && !JetpackFeaturesRemovalCoordinator.shouldRemoveJetpackFeatures()
+    }
```

Comment:
I noticed that we weren't using the same conditions for determining if the Story Post action should be shown in different places. For this reason, I decided to unify the logic into a single function. Specifically, checking if the device is an iPad. Based on [this PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/15996) and [this Kanvas issue](https://github.com/tumblr/kanvas-ios/issues/104), I understand that the Story post shouldn't be enabled on iPad.